weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/HumanFeedback/FeedbackSidebar.tsx

```diff
@@ -0,0 +1,226 @@
+import {toast} from '@wandb/weave/common/components/elements/Toast';
+import {useViewerInfo} from '@wandb/weave/common/hooks/useViewerInfo';
+import {Button} from '@wandb/weave/components/Button';
+import {Icon} from '@wandb/weave/components/Icon';
+import {makeRefCall} from '@wandb/weave/util/refs';
+import React, {useState} from 'react';
+
+import {HumanFeedbackCell} from './HumanFeedback';
+import {tsHumanAnnotationSpec} from './humanFeedbackTypes';
+
+type FeedbackSidebarProps = {
+  humanFeedbackSpecs: tsHumanAnnotationSpec[];
```

Comment:
here, and everywhere else, i think you want to rename these to remove "human" and emphasize "annotation"